hw/ip/prim_generic/lint/prim_generic_clock_gating.waiver

```diff
@@ -7,3 +7,5 @@ waive -rules LATCH                -location {prim_generic_clock_gating.sv} -rege
       -comment "clock gating cell creates a latch"
 waive -rules COMBO_NBA            -location {prim_generic_clock_gating.sv} -regexp {Non-blocking assignment to 'en_latch'} \
       -comment "clock gating cell creates a latch"
+waive -rules PARAM_NOT_USED -location {prim_generic_clock_gating.sv} -regexp {Parameter 'NoFpgaGate' not used.*} \
```

Comment:
Nit: The regexes in AscentLint are not anchored, you can remove the `.*` at the beginning/end (here and in other waivers)